src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -568,3 +576,21 @@ def _collect_control_machine_os_level_info(self):
         
         os_data["path_stats"] = file_stats
         self.env_info_dict["os_data"] = os_data
+
+    def _collect_service_status_info(self):
+        """Collects service information for each service in SERVICE_NAMES. This method changes the
+        ansible context for each check so that ansible logs the status to a service-specific file.
+        """
+        self.service_status_directory = os.path.join(self.curr_audit_directory, "service_statuses")
+        self._ensure_dir_exists(self.service_status_directory)
+
+        for service_name in SERVICE_NAMES:
```

Comment:
You can ignore one of `elasticsearch` or `elasticsearch-classic` since they both do the same status check (`elasticsearch` calls `elasticsearch-classic` for status checks).